cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -56,8 +56,7 @@ spec:
           timeoutSeconds: 30
       volumes:
       - name: kubernetes-dashboard-certs
-        secret:
-          secretName: kubernetes-dashboard-certs
+        emptyDir: {}
```

Comment:
Don't merge this change. We require the dashboard bump and revert of this before merging into a release branch